build.sbt

```diff
@@ -24,10 +24,14 @@ lazy val rebarFirrtl = (project in file("tools/firrtl"))
 
 lazy val rocketchip = RootProject(file("generators/rocket-chip"))
 
-lazy val testchipip = (project in file("generators/testchipip"))
+lazy val rebarrocketchip = project
```

Comment:
What does this achieve?

Reply:
This is to allow people to build directly from the `rocket-chip` repo. 

For example, you want to use their config, harness, generator, etc. So you can set all the make params to address this (use `freechips.rocketchip.system` etc). However, you also need to point to the SBT project that the generator is in. In this case, I was unable to use the `rocketchip` SBT project (assuming this is something with `RootProject` not allowing you to access SBT projects inside of it). So I did the Hwacha approach of having a simple project that only depends on `rocketchip` so that SBT doesn't have to rebuild the other projects in `example`. The cleaner solution is to just to use `example` but if `testchipip` changes you have to have SBT rebuild it when you don't want it in this case.

Reply:
Ah, curious.

Reply:
Any thoughts? Does `firechip` care about this extra SBT project?